tests/suites/constraints/constraints_aws.sh

```diff
@@ -0,0 +1,35 @@
+run_constraints_aws() {
+  # Echo out to ensure nice output to the test suite.
+  echo
+
+  file="${TEST_DIR}/constraints-aws.txt"
+
+  ensure "constraints-aws" "${file}"
+
+  echo "Deploy 3 machines with different constraints"
+  juju add-machine --constraints "root-disk=16G"
+  juju add-machine --constraints "cores=4 root-disk=16G"
+  juju add-machine --constraints "instance-type=t2.nano"
```

Comment:
this is aws specific so we can just drop it

Reply:
Ok, are we sure that we wish to drop aws specific test? 

If so, I will rename 'lxd' --> 'containrer', and 'aws' --> 'vm'. 
And further will allow 'vm' on all non-lxd and non-k8s substrates.